pkgs/applications/virtualization/singularity/default.nix

```diff
@@ -46,24 +49,23 @@ buildGoPackage rec {
 
     # Don't install SUID binaries
     sed -i 's/-m 4755/-m 755/g' builddir/Makefile
-
   '';
 
   buildPhase = ''
+    runHook preBuild
     make -C builddir
+    runHook postBuild
   '';
```

Comment:
```suggestion
  makeFlags = [ "-C" "builddir" ];
```

Reply:
This doesn't work here as the buildPhase for go packages is a "go build" not a "make".